src/ts/components/subtitleoverlay.ts

```diff
@@ -54,6 +56,12 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
         event.position.column = event.position.column || 0;
       }
 
+      if (event.region) {
```

Comment:
Why don't you check for `regionStyle` here? Seems odd given that you only use the `regionStyle` propety on the label. Also, does it make sense to have a `region` specified, remove the default positioning, and then not have a `regionStyle` for the label?

Reply:
You are right, checking for `regionStyle` actually makes more sense.